tests/test_examples.py

```diff
@@ -90,3 +110,13 @@ def test_cuda(example):
     filename, args = example[0], example[1:]
     filename = os.path.join(EXAMPLES_DIR, filename)
     check_call([sys.executable, filename] + args)
+
+
```

Comment:
Can we also add a `skipif` on `CI` for now? The examples stage takes quite a bit of travis time.